### ISSUE TITLE
Discussion about network identifiers

### ISSUE THREAD

**Issue by TimoGlastra** [hyperledger/indy-did-networks#3]
This issue is to start a discussion about network identifiers for indy networks. The governance hasn't been set-up, but as people are starting to use did:indy, it's important to open the discussion on which identifiers should be used for the existing networks.

The universal resolver has defined [their own mapping](https://docs.trinsic.id/reference/proto/#services-provider-v1-IndyOptions-IndyNetwork), and I think Trinsic's mapping is based on that (@tmarkovski is that correct?)

The universal resolver defines the following networks:

- Sovrin Mainnet - `sovrin`
- Sovrin Buildernet - `sovrin:builder`
- Sovrin Stagingnet - `sovrin:staging`
- Danube - `danube`
- IDunion - `idunion`
- IDunion Test - `idunion:test`
- Indicio Main - `indicio`
- Indicio Test - `indicio:test`
- Indicio Demo - `indicio:demo`
- `bbu` -- I don't know this network?

In addition we should probably create an identifier for BCovrin (`bcovrin` or `bcovrin:test`?).

Based on  https://github.com/decentralized-identity/uni-resolver-driver-did-indy/blob/main/docker/Dockerfile#LL39C45-L39C380

Any networks that should be added?

CC @swcurran @lynnbendixsen @tmarkovski @peacekeeper @ntsbs @paulbastian

**Comment by swcurran** [comment]
Thanks @TimoGlastra — I’m long overdue in getting this repository populated.

I believe that we will want to add CANdy to the list, right @WadeBarnes ?

Let’s go with BCovrin as being the root (even though there will likely never be a BCovrin “MainNet”) and `bcovrin:test`. I think there is also a `bcovrin:dev` that we may or may not add…

I’ve long had on my list to create the documents for this repo and will see if I can get a pull request together for this. I’d like this site published with a static site generator (docusaurus or Mkdocs Material) as well.

**Comment by paulbastian** [comment]
Looks fine to me

**Comment by WadeBarnes** [comment]
CANdy Networks (pan-Canadian Ledgers):
- CANdy-Dev - `candy:dev`
- CANdy-Test - `candy:test`
- CANdy-Prod - `candy`

BCovrin Networks (popular dev/test ledgers hosted by BC Gov):
- dev.greenlight.bcovrin - `bcovrin:dev.greenlight`
- greenlight.bcovrin - `bcovrin:greenlight`
- dev.bcovrin - `bcovrin:dev`
- test.bcovrin - `bcovrin:test`
- prod.bcovrin - `bcovrin`

@swcurran, We want to list the CANdy ledgers for sure.  I've listed all of the BCovrin networks, I'm not sure we'd want to list the `greenlight` ones, though they are listed in some educational materials.  Also `prod.bcovrin` does not get much use either.  It was more for us to use as a placeholder for Sovrin MainNet when we first started with the OrgBook.

**Comment by peacekeeper** [comment]
> The universal resolver has defined their own mapping

@TimoGlastra You're correct that for the Universal Resolver default configuration, we basically "made up" a list of networks that we had been familiar with. But it would be much better if HL Indy provided some governance and an authoritative mapping from network names to pool config files, and then Universal Resolver and other projects should just follow that.

Another place where Indy network names have been configured is this IDunion repository: https://github.com/IDunion/indy-did-networks/tree/main/networks

Some additional thoughts/comments:
- For IDunion, I have seen different examples over time, sometimes using `idunion`, and sometimes `idu`. But I think there's consensus around `idunion`.
- `bbu` is Bedrock Consortium, driven by IBM: https://bedrockconsortium.org/. But not sure how active it is at the moment. Maybe @vinomaster can comment?
- What to do if names change? For example the Sovrin Board has decided to rename "StagingNet" to "TestNet", and to make "BuilderNet" a purely internal network that will not be open to the public anymore.

**Comment by c2bo** [comment]
I didn't see the discussion here, sorry for the late response.
The IDunion networks repository was always meant to be moved into this one once we have a bit of a governance framework set up (who registers networks, how do we update the genesis files etc.) - it was a testing ground for the indy-vdr update (did:indy) before the networks repository was created here.

**Comment by PatStLouis** [comment]
is someone working on a draft PR to include the networks/genesis files? Happy to do the work if time is the issue...

**Comment by lynnbendixsen** [comment]
@PatStLouis  I was going to get started on this but haven't found the time yet.  If you would like to start in on the draft that would be great!

**Comment by PatStLouis** [comment]
Sounds good, I will get something in this week. I will use the link provided by @jleach as a starting point to populate the genesis data and follow the directory structure from the iDunion repository. We can take it from there.

**Comment by PatStLouis** [comment]
Here's a draft PR #5
